js/summary.go

```diff
@@ -117,6 +117,17 @@ func summarizeMetricsToObject(data *lib.Summary, options lib.Options) map[string
 	}
 	m["metrics"] = metricsData
 
+	var setupDataI interface{}
+	if setupData != nil {
+		if err := json.Unmarshal(setupData, &setupDataI); err != nil {
+			return nil
```

Comment:
hmm I don't like that we're silently ignoring the error here, _and_ returning nothing at the same time :confused: this error is very, _very_ unlikely to ever happen, but if it does, we should still at least return `m` instead of `nil` here

Reply:
Makes sense. Can I just make it return m?

Reply:
Yep, that is what I meant. Maybe add a `// TODO: log the error` as well, though since it probably won't ever occur, it's not a big deal.